ledger/ledger.go

```diff
@@ -10,8 +10,8 @@ import (
 	ledgerPb "github.com/tron-us/go-btfs-common/protos/ledger"
 	"github.com/tron-us/go-common/v2/log"
 
-	"github.com/gogo/protobuf/proto"
 	ic "github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/tron-us/protobuf/proto"
```

Comment:
Does this mean that all projects need to import this version, or will the gogo version be compatible? (since you only modified the generator, not the actual lib)